src/js/modal-window.js

```diff
@@ -148,7 +153,9 @@ function updateLibraryButton(movieId) {
 
   if (isInLibrary) {
     libraryBtn.textContent = 'Remove from my library';
+    libraryBtn.dataset.action = 'remove';
```

Comment:
szukałem tego dataset gdzie jest używany ale nie widzę
jak niepotrzebny to do wywalenia